pkg/network/config/config_linux.go

```diff
@@ -37,10 +38,12 @@ func (c *Config) EnabledProbes(runtimeTracer bool) (map[probes.ProbeName]struct{
 			enabled[probes.TCPRetransmit] = struct{}{}
 		}
 
-		enabled[probes.SockFDLookup] = struct{}{}
-		enabled[probes.SockFDLookupRet] = struct{}{}
-		enabled[probes.DoSendfile] = struct{}{}
-		enabled[probes.DoSendfileRet] = struct{}{}
+		if ebpf.VerifyKernelFuncs("sockfd_lookup_light") {
```

Comment:
I guess this mean our `sendfile` fix will be no more on GCP?

Reply:
For now. There isn't an easy/fast fix. We are going to assess options for 7.31.